internal/views/authors/new.html

```diff
@@ -0,0 +1,17 @@
+<!DOCTYPE html>
+<html lang="pt-br">
+<head>
+    <meta charset="UTF-8">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>Novo Autor</title>
+
+</head>
+<body>
+    <h3>Novo Autor</h3>
+    <form method="post" action="/authors">
+        <label for="name">Nome</label>
+        <input type="text" name="name" required>
```

Comment:
```suggestion
        <input type="text" id="name" name="name" required>
```

O campo de `input` precisa de um `id` para o `for` do `label` funcionar.